Source/Core/DolphinQt2/Host.cpp

```diff
@@ -105,6 +106,7 @@ void Host_UpdateDisasmDialog()
 
 void Host_UpdateProgressDialog(const char* caption, int position, int total)
 {
+  emit Host::GetInstance()->UpdateProgressDialog(QString::fromStdString(caption), position, total);
```

Comment:
Are the strings returned by translation UTF-8? Should this be QString::fromUtf8/QString::fromLocal8Bit? also avoids the temporary std::string.